arbor/simulation.cpp

```diff
@@ -220,6 +220,7 @@ time_type simulation_state::run(time_type tfinal, time_type dt) {
     // If spike exchange and cell update are serialized, this is the
     // minimum delay of the network, however we use half this period
     // to overlap communication and computation.
+    PI(task_system_);
```

Comment:
This should be done in the calling code.

The profiler is turned on/off, started/stopped etc by the calling code. In our case this will be in the miniapps.